src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/Contracts/DetachDatabaseRequest.cs

```diff
@@ -16,6 +16,10 @@ public class DetachDatabaseRequestParams : GeneralRequestDetails
         /// </summary>
         public string ObjectUrn { get; set; }
         /// <summary>
+        /// The target database name.
+        /// </summary>
+        public string Database { get; set; }
```

Comment:
Have to pass in the database name for these operations so that we generate the same connection string as the pool with the lingering db connections.